### ISSUE TITLE
Error during beginner tutorial

### ISSUE THREAD

**Issue by AliciaMacdonald** [haskell-webdriver/haskell-webdriver#126]
I'm having a bit of trouble getting the beginner tutorial to work. I have installed the selenium server and it's running on my OSX laptop at `http://127.0.0.1:4444/`. It's also currently working fine with the Python 3 bindings, so I'm thinking it must be something I've done wrong while trying to get the Haskell bindings to work.

When I run `main` from the beginner tutorial I get the following error:
```
*** Exception: UnknownCommand "Request {
  host                 = "127.0.0.1"
  port                 = 4444
  secure               = False
  requestHeaders       = [("Accept","application/json;charset=UTF-8"),("Content-Type","application/json;charset=UTF-8"),("Content-Length","130")]
  path                 = "/wd/hub/session"
  queryString          = ""
  method               = "POST"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}"
```

It doesn't even open Firefox or anything... tried a whole bunch of stuff to try and figure out what was wrong. I still get the exact same error even if I write:

```
main = runSession defaultConfig $ return ()
```

Any ideas?

**Comment by ptek** [comment]
Hi Ethan,

It looks like you are trying to use Selenium 3. I had similar problems recently.
Could you see if Selenium 2.* version works for you?

Pavlo.

**Comment by kallisti-dev** [comment]
Yes. Newer versions of selenium use an updated REST API that we do not
support yet.

**Comment by brezal** [comment]
@ptek @kallisti-dev I experience this same issue with selenium-server-standalone-2.53.1.jar

**Comment by codygman** [comment]
Am I right in assuming these bindings start up the selenium server for me? I'm guessing not. Did you also assume this @EthanMacdonald ?

**Comment by codygman** [comment]
@kallisti-dev Not sure if you use nixos or the nix package manager, but if so here's a reproducible example you can use from `nix-shell`:

https://github.com/codygman/webdriver-nixos-haskell-example

**Comment by codygman** [comment]
Okay this is now working with 2.53.1 for me as of https://github.com/codygman/webdriver-nixos-haskell-example/tree/f705d40e9fedb1c3122aef5955ef096cb4497546

**Comment by kallisti-dev** [comment]
@codygman  I'm thinking Nix could be a tool to mention in the README, and to integrate with the project itself. It would simplify a lot of the installation process (no need to go and manually install chromedriver)

However I'm not very familiar with Nix myself, so I will need some time to play around with it. Do you know if we could use it in conjunction with Travis CI to generate test builds for different versions of Selenium standalone and chromedriver? Might need to use SauceLabs at that point..

**Comment by thomasjm** [comment]
We don't provide examples/tutorials anymore, in favor of tests and easy to understand docs, so I'm going to close this.

The tests use Nix to obtain things like Selenium and chromedriver. For a more batteries-included approach that sets everything up for you, I'd recommend [sandwich-webdriver](https://hackage.haskell.org/package/sandwich-webdriver).
